src/__tests__/CitySearch.test.js

```diff
@@ -55,6 +55,13 @@ describe('<CitySearch /> component', () => {
         expect(CitySearchWrapper.state("suggestions")).toEqual(filteredLocations);
     });
 
+    //NEWLY ADDED TEST
+    test('when the suggestion input does not contain any character', async () => {
+        const eventObject = { target: { value: '' } };
+        await CitySearchWrapper.find('.CitySearchInput').simulate('change', eventObject);
```

Comment:
There is no input whose className is equal to `.CitySearchInput`. Please use another value to select the input.